pkg/config/draftconfig.go

```diff
@@ -20,26 +21,24 @@ type FileNameOverride struct {
 }
 
 type BuilderVar struct {
-	Name             string   `yaml:"name"`
+	DefaultValue     string   `yaml:"default"`
 	Description      string   `yaml:"description"`
-	VarType          string   `yaml:"type"`
 	ExampleValues    []string `yaml:"exampleValues"`
-}
-
-type BuilderVarDefault struct {
-	Name             string `yaml:"name"`
-	Value            string `yaml:"value"`
-	ReferenceVar     string `yaml:"referenceVar"`
-	IsPromptDisabled bool   `yaml:"disablePrompt"`
+	IsPromptDisabled bool     `yaml:"disablePrompt"`
```

Comment:
I love the change of grouping the default under here.

Grouping the fields relevant to the Default together still could make sense here to help make this more digestible. For example, in its current form `IsPromptDisabled` specifically refers to when defaulting, its context is completely related to that so it makes sense to have it in a sub struct.

Reply:
That's a good point. At the moment though I would say that due to the integration of DevHub functionality, there's one envArg in particular that I'm using IsPromptDisabled for beyond its original purpose of being just for default variables. The envArg is DOCKERFILE. Draft's original templates don't include this arg, but DevHub does, and the issue I have with asking the Draft user to provide it is that in order for it to work on the template it needs to come after a "-f" flag indicator. I don't want this "-f" to be a literal on the template in the case that the user doesn't provide a DOCKERFILE and at the moment I haven't made the proper changes to how we prompt in Draft to handle this specific case. So, the way I'm currently getting around it is by including the "-f" within the envArg's value on DevHub and then using IsPromptDisabled on Draft so that it resorts to using its default value of an empty string (and its default val being an empty string defeats the purpose of including DOCKERFILE as a BuilderVarDefault{}).

Reply:
Figured out that a good default value for DOCKERFILE would be "./Dockerfile" so I'm going to go ahead and create a substruct specifically for defaults holding the defaultVal and IsPromptDisabled.